stamp_generation/stamp_templates/stamp_template1.html.jinja

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
```

Comment:
Why 1 in the name?

Reply:
I was planning to have multiple templates earlier. But since it's only one template for now I have removed 1 from the name.